tools/who_what_benchmark/whowhatbench/wwb.py

```diff
@@ -194,6 +194,12 @@ def parse_args():
         help="LLMPipeline specific parameter that defines the use of a long context prompt",
     )
 
+    parser.add_argument(
+        "--empty_adapters",
+        action="store_true",
+        help="Inference without adapters",
```

Comment:
smth like `Inference with empty adapters. Applicable for GenAI text-to-image task.`
If I understand correctly, the user can run it as is and it will be a mode without adapters.